src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+
+import "./App.css";
+import { PasswordInput } from "./PasswordInput";
+import { PasswordIndicatorGrid } from "./PasswordIndicatorGrid";
+
+function App() {
+  const [password, updatePassword] = useState("");
```

Comment:
using [react hooks](https://reactjs.org/docs/hooks-intro.html) here so we can write our components and functions instead of objects, lmk if there's any questions about that